helm/logging-operator/templates/configmap.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "resource.default.name"  . }}
+  namespace: {{ include "resource.default.namespace"  . }}
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+data:
+  config.yml: |
+    server:
```

Comment:
This config looks like an operator kit one :)

Reply:
Should I change some things in it ?

Reply:
I'm not sure, let's get this running and change what needs to be changed :p

Reply:
True, logging-operator currently does not support config files. You can remove the whole volume and its references in `args`.
However, we'll have to set some `args` depending on some config values.